Places/Scenes/CreatePlace/CreatePlaceInteractor.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  CreatePlaceInteractor.swift
+//  Places
+//
+//  Created by Andressa Valengo on 24/01/25.
+//  Copyright (c) 2025 ___ORGANIZATIONNAME___. All rights reserved.
+//
+
+import UIKit
+
+protocol CreatePlaceBusinessLogic {
+  var placeToEdit: Place? { get }
+  func isPlaceNameValid(name: String) -> Bool
+  func createPlace(request: CreatePlace.CreatePlace.Request)
+  func showPlaceToEdit(request: CreatePlace.EditPlace.Request)
+  func updatePlace(request: CreatePlace.UpdatePlace.Request)
+}
+
+protocol CreatePlaceDataStore {
+  var placeToEdit: Place? { get set }
+}
+
+class CreatePlaceInteractor: CreatePlaceBusinessLogic, CreatePlaceDataStore {
+  
+  var presenter: CreatePlacePresentationLogic?
+  
+  var placesWorker = PlacesWorker(placesStore: PlacesMemStore())
```

Comment:
```suggestion
  private var presenter: CreatePlacePresentationLogic?
  
  private var placesWorker = PlacesWorker(placesStore: PlacesMemStore())
```
Essas variáveis não deveriam poder ser acessadas externamente, considere adicionar o private